modules/virt-temporary-token-VNC.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+:_content-type: PROCEDURE
+[id="virt-temporary-token-VNC_{context}"]
+= Generating a temporary token for the VNC console
+
+Generate a temporary token to access the VNC of a virtual machine (VM).
+
+.Prerequisites
+
+* A running virtual machine with {VirtProductName} 4.14 or later and xref:../../virt/about-virt/virt-architecture#virt-about-ssp-operator_virt-architecture[`ssp-operator`] 4.14 or later
+
+.Procedure
+
+. Enable the feature gate in the HyperConverged (`HCO`) custom resource (CR):
++
+[source,yaml]
+----
+$ oc patch hyperconverged kubevirt-hyperconverged -n {CNVNamespace} \
+ --type json -p '[{"op": "replace", "path": \
+ "/spec/featureGates/deployVmConsoleProxy", \
+ "value": true}]'
+# ...
+----
+
+. Generate a token:
++
+[source,terminal]
+----
+$ curl --header "Authorization: Bearer ${TOKEN}" \
+     "https://api.<cluster_fqdn>/apis/token.kubevirt.io/v1alpha1/namespaces/<namespace>/virtualmachines/<vm_name>/vnc?duration=<duration>" <1>
+----
+<1> Duration can be in hours and/or minutes. Example: `5h30m`
++
+Sample output:
++
+[source,terminal]
+----
+{ "token": "eyJhb..." }
+----
+
+. Optional: Use the token provided in the output to create a variable:
++
+[source,terminal]
+----
+$ export VNC_TOKEN="<token>"
+----
+
+.  Log in to the cluster:
++
+[source,terminal]
+----
+$ oc login --token ${VNC_TOKEN}
+----
+
+You can now use the token to access the VNC console of a VM.
```

Comment:
I would move these lines to the `Verification` part below. `oc login` is only needed if the token is used by `virtctl vnc`.